### ISSUE TITLE
Recording to self-contained, executable bash file

### ISSUE THREAD

**Issue by sickill** [asciinema/asciinema#166]

**Comment by cypa** [comment]
what exactly do you meant?
produce portable executable containg record to play the record upon executing itself?

**Comment by sickill** [comment]
Yes. It could basically be a generated bash file with only `echo -e -n "..."; sleep ...; echo -e -n "..."; sleep ` statements.
